examples/type-exclusion/README.md

```diff
@@ -1 +0,0 @@
-When a particular type is disallowed (but there are no other constraints), we should see sample data for other, non-forbidden types. To run this profile it is necessary to use the command line option `--allow-untyped-fields`.
```

Comment:
Looks like the `--allow-untyped-fields` command will no longer do anything. If so, it needs to be removed. It might not need doing in this PR, but you should at least check that it doesn't crash if you use it.

Reply:
deffo delete it